docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -610,88 +620,86 @@ The table below shows the treatment category coverage proportions for the isatux
      - 30.6%
    * - 3
      - Isatuxamib-containing
-     - 1.3%
-     - 10%
-     - 10%
+     - 1.0%
+     - 9%
+     - 9%
    * - 3
      - Daratumuab-containing
      - 25.6%
      - 46.3%
      - 46.3%
    * - 4
      - Isatuxamib-containing
-     - 1.5%
-     - 10%
-     - 10%
+     - 3.3%
+     - 7%
+     - 7%
    * - 4
      - Daratumuab-containing
      - 29.7%
      - 53.7%
      - 53.7%
    * - 5+
      - Isatuxamib-containing
-     - 0.9%
-     - 10%
-     - 10%
+     - 3.3%
+     - 7%
+     - 7%
    * - 5+
      - Daratumuab-containing
      - 17.1%
      - 30.9%
      - 30.9%
 
-*Initialization*:
-
-	Each simulant initialized into a multiple myeloma state should be assigned to a treatment category. The probability of being assigned to a given category should be equal to the coverage for that category in 2021 that is specific to the line of treatment corresponding to the MM cause model state that the simulant occupies. We will assume that at initialization, no simulants in the isa/dara-containing treatment categories have been retreated with an anti-CD38 monoclonal antibody treatment.
+*Burn-in period initialization*:
 
-	If a simulant is assigned to the isatuxamib-containing treatment category or the daratumumab-containing treatment category, they should be assigned the attribute :code:`eligible_for_anticd38_retreatment = True`; otherwise, set :code:`eligible_for_anticd38_retreatment = NaN`.
+	Initialization for the treatment burn-in period should occur prior to the introduction of anti-CD38 monoclonal antibody treatments on Jan. 1, 2016. Each simulant should be initialized to the residual treatment category. Each simulant should be initialized to :code:`prior_anticd38_treatment = False`. Initialization on Jan. 1, 2016 should include patients in all of the relapsed and refractory states, not only simulants in the newly diagnosed MM state.
 
-	NOTE: This strategy makes several assumptions, including:
+*Post-burn in period initialization*:
 
-		- Incident treatment coverage is equal to prevalent treatment coverage of isatuxamib. In other words, it assumes that there is similar progression-free survival across treatment categories.
+  Post-burn in period initialization should occur at the official start of the simulation time frame on Jan. 1, 2021. Simulants should be initialized according to the treatment line-specific treatment category prevalence and :code:`prior_anticd38_treatment` status prevalence on Jan. 1, 2021 from the burn-in period run as described above.
```

Comment:
I have some questions/issues to work out about how to implement this, but this sounds feasible.

Reply:
Ok, just let me know if you want to talk it over!